<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student_page</class>
 <widget class="QDialog" name="student_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1410</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="details">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>CLASSMATES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1234</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MODIFY ACCOUNT</string>
   </property>
  </widget>
  <widget class="QListWidget" name="db_list">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>611</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>MY COURSE</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_student">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="student_pic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>600</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QWidget" name="password_w" native="true">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>50</y>
     <width>501</width>
     <height>511</height>
    </rect>
   </property>
   <widget class="QLabel" name="password_pic">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string> MODIFY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>NEW_PASSWORD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_password">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 100%;
  border: 2px solid #ccc;
  color:black;
  font-size: 16px;
  background-color: white;
  background-position: 10px 10px; 
  background-repeat: no-repeat;
</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MODIFY</string>
    </property>
   </widget>
  </widget>
  <zorder>password_w</zorder>
  <zorder>details</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>db_list</zorder>
  <zorder>pushButton</zorder>
  <zorder>id_student</zorder>
  <zorder>student_pic</zorder>
  <zorder>clear</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
